src/commands/registerCommands.ts

```diff
@@ -18,8 +18,12 @@ import { getAgentBenchmarkConfigs, getCommands, runWizardCommandWithInputs, runW
 import { ext } from '../extensionVariables';
 import { installOrUpdateFuncCoreTools } from '../funcCoreTools/installOrUpdateFuncCoreTools';
 import { uninstallFuncCoreTools } from '../funcCoreTools/uninstallFuncCoreTools';
+import { type DurableTaskSchedulerClient } from '../tree/durableTaskScheduler/DurableTaskSchedulerClient';
```

Comment:
You really just went and reorganized this whole file huh?

Reply:
💩